---
author: Emilie Boulanger, Saara Suominen
excerpt: OBIS is actively engaged in eDNA research and the eDNA research community.
  Through its many projects, OBIS develops monitoring studies as well as tools for
  the bioinformatic processing and analysis of eDNA data, and works on the development
  of community-approved metadata standards. As a data publication platform, OBIS enables
  research teams, authors, and monitoring projects to publish occurrence data derived
  from quantitative (qPCR/ddPCR) or community-level (metabarcoding) eDNA approaches
  to make ocean biodiversity data globally accessible.
feed: true
identifier: obis-edna
image: /images/obis_edna_thumb.png
lang: en
layout: post
purpose: news
tags:
- OBIS
- eDNA
title: OBIS eDNA services, expertise and data publication
---

# OBIS eDNA services, expertise and data publication

- [Introduction to eDNA and OBIS](##Overview)
- [Publishing eDNA data to OBIS](##Publishing-eDNA-data-to-OBIS)
- [Accessing eDNA data from OBIS](##Accessing-eDNA-data-from-OBIS)
- [eDNA services and resources developed by OBIS](#edna-services-and-resources-developed-by-obis)
    - [Citizen Science eDNA Sampling Protocols](#citizen-science-edna-sampling-protocols)
    - [Bioinformatics pipeline : from raw sequences to species-annotated data in OBIS](#bioinformatics-pipeline--from-raw-sequences-to-species-annotated-data-in-obis)
    - [Analytical tools](#analytical-tools)
    - [Capacity building and training](#capacity-building-and-training)
- [OBIS projects developing eDNA research, tools and standards](#obis-projects-developing-edna-research-tools-and-standards)


## Overview
### Introduction to eDNA
Environmental DNA, or eDNA, refers to the genetic material that can be extracted from an environmental sample such as (sea)water, sediment, soil, or even air. This genetic material can be analysed with a set of molecular tools to infer information about the organisms living in the sampled environment. This DNA was shed as waste, mucus, or cells and the analyses vary from presence/absence detection with taxon-specific genetic markers, to the characterization of whole communities or even multi-species assemblages with broad-use markers. Sampling and analysing environmental DNA allows us to study the diversity of oceanographic life from its smallest fraction invisible to the naked eye to its largest inhabitants that are nonetheless challenging to survey, and all the organisms that fall in between.


The cost effective, ethical nature of eDNA sampling has the potential to revolutionise our knowledge about ecosystems and species diversity, and unlock new insights for biodiversity studies, environmental monitoring, as well as provide reliable data to support decision-making. The potential for standardisation with well-documented protocols and supporting metadata represent an important step in generating and sharing biodiversity data that is globally accessible, comparable and reusable. 

### Summary of OBIS eDNA services 
OBIS, the Ocean Biodiversity Information System, is actively engaged in eDNA research and the eDNA research community. Through its many projects, OBIS develops tools for the bioinformatic processing and analysis of eDNA data, works on the development of metadata standards, and can provide services to support partners in field sampling and lab work, and in making eDNA data findable, accessible, interoperable, and reusable (FAIR).

As a data publication platform, OBIS provides rigorous quality control and can publish occurrence data derived from quantitative (qPCR/ddPCR) or community-level (metabarcoding) eDNA approaches following community-approved data standards. This data can in turn be queried and accessed from OBIS to increase the reach and application of eDNA-based biodiversity data. The associated metadata not only provides the essential sampling information (longitude & latitude coordinates, depth, …) but allows publishers to link critical information on the environment sampled, the different protocols and methods used across the workflow (target gene, primer sequences, clustering or denoising approach, reference database used, …), the DOI of the published study the data comes from, as well as link back to the original sequence data deposited in INSDC databases (e.g. EMBL-EBI’s ENA or NCBI’s SRA).

## Publishing eDNA data to OBIS


The guidebook on [Publishing DNA-derived data through biodiversity data platforms](https://docs.gbif.org/publishing-dna-derived-data/en/){:target="_blank"}, co-authored by OBIS and GBIF, provides a comprehensive view of why, where and how to publish DNA-derived biodiversity data. 
The [OBIS manual](https://manual.obis.org){:target="_blank"} further provides all the information needed to publish biodiversity data to OBIS, with a section dedicated to [publishing DNA-derived data](https://manual.obis.org/dna_data.html){:target="_blank"}. 

The data published to OBIS follows the Darwin Core community data format, ensuring the data is interoperable across many databases, searchable, accessible and machine-readable. A detailed explanation of the terms are readily available in the [Guidelines Document - 2.2 Data Mapping](https://docs.gbif.org/publishing-dna-derived-data/en/#data-mapping) and the dedicated [DwC dnaderivdedata extension](https://rs.gbif.org/extension/gbif/1.0/dna_derived_data_2022-02-23.xml) page. The OBIS resources page hosts a tool for [finding the right DwC format](https://resources.obis.org/find-your-dwc/). In short, DNA-derived data is published to OBIS as an Occurrence core dataset supplemented with the DNA derived data extension. The occurrence table captures information such as the occurrence id, sampling date, location, time, ASV, read abundance, taxonomic assignment and the taxon's corresponding WoRMS ID. The dnaderiveddata table in turns captures DNA-specific information such as the target gene, primers used, sequencing platform, bioinformatic parameters and the ASV sequence. It again contains the occurrence id to link the data to the occurrence table.

A series of video tutorials are available on the [OBIS Youtube Channel](https://www.youtube.com/@oceanbiodiversityinformati6931) to help you prepare and publish your datasets. These include the series [How To Use & Publish data with OBIS](https://www.youtube.com/playlist?list=PLlgUwSvpCFS4TS7ZN0fhByj_3EBZ5lXbF) which explains how to format your datasets to follow DwC criteria. OBIS also organised a webinar on genetic data, with an introduction to how OBIS is incorporating data, how genetic data can be accessed and a use case from the first eDNA dataset provided by OBIS-USA. The recording of the webinar can be watched [here](https://www.youtube.com/watch?v=KKh_Hd8zybs).

Finally, OBIS has an example eDNA metabarcoding dataset with scripts for data formatting available on the [OBIS github page](https://github.com/iobis/dataset-edna). 

### OBIS vs GBIF: marine-specific guidelines tailored for OBIS

OBIS - the [Ocean Biodiversity Information System](https://obis.org/) — and GBIF — the [Global Biodiversity Information Facility](https://www.gbif.org/) are both global biodiversity data sharing platforms and have a [joint strategy](https://doi.org/10.35035/doc-kcqs-5h52) and [action plan](https://doi.org/10.35035/doc-e52v-5875) to ensure the cross-platform flows and services of high-quality data about marine and coastal biodiversity. Both platforms use the same guidelines and Darwin Core community data format, and the same [Integrated Publishing Toolkit (IPT)](https://manual.obis.org/ipt) to publish and register datasets. When publishing a dataset through either OBIS or GBIF nodes, selecting the option to register the data to GBIF or OBIS ensures that the data flows from one platform to the next.

OBIS is focused on the marine realm and marine taxa, whereas GBIF publishes data across both terrestrial and marine realms. With the focus on marine datasets, stringent quality controls on the data published to OBIS increase the reliability of the data and lead to small differences in what information is required for publishing in OBIS as opposed to GBIF. The two main differences consider the taxonomic backbone used and the geographic coordinates data checks:
- To ensure consistent taxonomic nomenclature OBIS uses the [World Register of Marine Species (WoRMS)](https://www.marinespecies.org/) as the only taxonomic backbone. This is the case also for occurrences derived from genetic data; a scientific name linked to a scientific name ID from the WoRMS database is highly recommended information for publishing. [...] Fully unclassified sequences are recommended to be classified as "incertae sedis", with the WoRMS _scientificNameID urn:lsid:marinespecies.org:taxname:12_. 
- Another required field in OBIS data submissions are geographic coordinates. OBIS performs additional quality checks related to marine data; e.g. that coordinates for strictly marine species are not on land, and that the depth value reported is in a reasonable range.

## Accessing eDNA data from OBIS

Data in general, and DNA-derived data specifically, can be queried and downloaded from OBIS through the online mapper or the R package robis. This is also covered in the OBIS manual under ["How to find genetic data in OBIS"](https://manual.obis.org/dna_data.html#how-to-find-genetic-data-in-obis).

- **Using the mapper**

The [mapper tool](https://mapper.obis.org) allows you to query biodiversity data on OBIS by searching for data using specific criteria such as scientific name or geographic area. 
eDNA-based biodiversity data can also specifically be searched for by specifying the option for [DNAderiveddata](https://mapper.obis.org/?hasextensions=DNADerivedData#) under the Extensions tab. As of the day of publishing this article, the resulting data layer holds **19,815,140 records** of **5,226 species** and **8,694 taxa** across 51 datasets.

Once this layer is added, the data can be downloaded either all together as Darwin Core Archive dataset, or you can access each published dataset separately and download the original Darwin Core formatted files. If the dataset originates from a published study, the doi of that paper is provided to link to all the necessary information.

 - **Data access tools (R, API)**

Genetic data can also be accessed from OBIS using the R package [robis](https://github.com/iobis/robis). Instructions for using the robis package to access DNA derived data can be found in the [OBIS manual](https://manual.obis.org/access.html#r-package) and the dedicated [vignette](https://iobis.github.io/notebook-dnaderiveddata/).

Both the mapper and the R package are based on the [OBIS API](https://api.obis.org/), which can also be used to find and download data. When [using the API](https://manual.obis.org/access.html#api) directly, you can filter the Occurrence records by specifying the extension DNADerivedData. You can further search through this e.g. by scientific name, date, depth, coordinates, or country.

### Examples of eDNA datasets hosted on OBIS

There are currently 51 eDNA datasets publicly available on OBIS, amounting to 19,815,140 records of 5,226 species and 8,694 taxa.

A few examples:
- [Environmental DNA reveals tropical shark diversity in contrasting levels of anthropogenic impact.](https://obis.org/dataset/dbb57c74-c34c-4502-b1d0-15324a7fabc4) 
This dataset has appeared in 3,643 downloads in 2024, with a total of 1,053,527 records.

- Confirmation of pygmy sperm whale, Kogia breviceps from Palk Bay, India through Cytochrome Oxidase1 (COI) sequence.   https://obis.org/dataset/f6db2e52-e89d-4e7b-950d-6001e54424c9

- Ecosystem biomonitoring with eDNA: Metabarcoding across the tree of life in a tropical marine environment. Coral Bay, Ningaloo Reef, Western Australia (2015)
https://obis.org/dataset/d71b6687-810e-4234-ae31-386017b5b08a   

- Marine zooplankton community structure through environmental DNA metabarcoding in the Florida Keys, collected by the South Florida Program (NOAA/AOML) and the Marine Biodiversity Observation Network (MBON)
https://obis.org/dataset/b41e10e2-5717-4bb3-b06a-7840a62aebcc

## eDNA services and resources developed by OBIS

A set of different tools and services were developed within the different eDNA projects led or supported by OBIS. These cover the different steps across an eDNA workflow, ranging from developing sampling protocols for citizen science, to developing bioinformatics pipelines to process raw eDNA sequences and enable the export of eDNA-derived biodiversity data ready to be uploaded to OBIS and accessed by users worldwide.

In addition to these services already developed, OBIS can guide partners across these steps in designing and carrying out eDNA studies as well as develop new tools depending on the project’s need.

### Citizen Science eDNA Sampling Protocols

As part of the [eDNA expeditions](https://www.unesco.org/en/edna-expeditions) project, the OBIS team collaboratively developed citizen science protocols for sampling eDNA. An instruction video, sampling booklets detailing the protocol in six languages and sample information sheets and infographics can be downloaded under the [eDNA expeditions
Training Materials](https://www.unesco.org/en/edna-training-materials?hub=66910). 


### Bioinformatics pipeline : from raw sequences to species-annotated data in OBIS

OBIS developed a bioinformatics pipeline for processing raw metabarcoding sequences under the [PacMAN project](https://pacman.obis.org). Broadly speaking, it creates a framework that receives raw sequence data from eDNA samples, cleans, aligns, classifies sequences, and finally outputs a DwC-compatible table. To note, the pipeline automatically searches for aphia-IDs from WoRMS to include in the DWC-compatible tables. In addition the output also contains a phyloseq object, which is compatible with the commonly used phyloseq R package for sequence data analysis. The pipeline is under active development. More details about the PacMAN pipeline can be found on its associated [GitHub repository](https://github.com/iobis/PacMAN-pipeline).


### Analytical tools
- robis   
    [robis](https://manual.obis.org/access.html#r-package) is OBIS’ flagship R package and a client for the OBIS API. It includes functions for data access, as well as a few helper functions for visualising occurrence data and extracting nested MeasurementOrFact or DNADerivedData records.

- obistools: Tools for data enhancement and quality control.  
    [obistools](https://iobis.github.io/obistools/) is an R package developed by OBIS collating a set of functions to check and map Darwin Core fields from a dataset and plot datapoints on a map.   

- Sequence Search Tool (in development)   
A prototype [Sequence Search tool](https://sequence.obis.org/) is in development that allows you to search for sequences or related sequences in OBIS. More info can be found in the [OBIS manual](https://manual.obis.org/dna_data.html#how-to-find-genetic-data-in-obis).

### Capacity building and training

OBIS provides training on eDNA sampling and analysis methods. These can e.g. be tailored towards scientists, practitioners, site managers or policy makers.   

A first training course was organised in Fiji in 2023 under the PacMAN project. The focus was on eDNA sampling, DNA extraction, qPCR for the detection of invasive species, and bioinformatic processing of metabarcoding data. All information and necessary links can be found on the PacMAN project website under the [“Marine Invasive Species Early Detection: Utilising Molecular Tools” training course page](https://pacman.obis.org/2023/04/21/pacman-scientific-training-course-online/).


## OBIS projects developing eDNA research, tools and standards
OBIS is currently leading two large-scale research and surveillance projects, engaging citizens globally to survey the biodiversity across 25 marine UNESCO World Heritage sites (eDNA expeditions) and developing information networks for the surveillance of marine invasive species in the Pacific (PacMANn). OBIS also participates in two Horizon Europe projects reviewing bioinformatic workflows (MARCO-BOLO), and rethinking the eDNA data publishing infrastructure and metadata standards (eDNAqua-Plan).


<img alt="eDNAexpeditions" src="/images/ednaexpeditions_kids.jpg" width="50%">

https://www.unesco.org/en/edna-expeditions


<img alt="pacman-logo" src="/images/pacman_logo.png" width="50%">

https://pacman.obis.org/

<img alt="ednaquaplan-logo" src="/images/ednaquaplan_logo.png" width="50%">

https://ednaquaplan.com/

<img alt="marcobolo-logo" src="/images/marcobolo_logo.png" width="50%">

https://marcobolo-project.eu/